apps/charterafrica/src/payload/blocks/ConsultationDocuments.js

```diff
@@ -0,0 +1,84 @@
+const ConsultationDocuments = {
+  slug: "consultation-documents",
+  fields: [
+    {
+      name: "documents",
```

Comment:
I think `group` is a better name since we're asking for group details i.e. name and options.